VirtualFilesystemTestTrait.php

```diff
@@ -88,7 +90,14 @@ public function providerTestData() {
 	 * Loads the configuration for the vfs structure and test data.
 	 */
 	protected function loadConfig() {
-		$this->config = require $this->getPathToFixturesDir() . $this->path_to_test_data;
+		$this->config = array_merge(
+			[
+				'vfs_dir'            => '',
+				'structure'          => [],
+				'test_data'          => [],
+			],
+			require $this->getPathToFixturesDir() . $this->path_to_test_data
+		);
```

Comment:
@Screenfeed I added a default structure to ensure each `$this->config` has the minimum required keys in it. What do you think?

Reply:
@hellofromtonya Yes, very good idea, more elegant than what I did.